### ISSUE TITLE
ValueError with example_bam

### ISSUE THREAD

**Issue by Eugene108** [betteridiot/bamnostic#48]
I get "ValueError: path is on mount 'C:', start on mount 'D:' " as error when I attempt to execute the following:

bam = bs.AlignmentFile(bs.example_bam, 'rb')

What am I missing here? I am running under Win 10. Thanks!

**Comment by betteridiot** [comment]
Please use the bug template when creating an issue. It should prompt me with everything I need to track the bug down.

As it stands, the error below
```python-traceback
ValueError: path is on mount 'C:', start on mount 'D':
```
is not a `bamnostic` error. It is an OS error. It is caused by the unique nature of how Windows creates it volumes. Whereas Linux-based environments have a root directory that allows one to traverse all volumes/drives, Windows does not. It has unique root directories for each volume/drive. (source: https://bugs.python.org/issue7195#msg94744)

Basically, the error is caused by having `bamnostic` installed in your C drive but your current working directory is your D drive. `bamnostic` tries to point to the `bs.example_bam` (which works fine), but the index file for it is built on relative pathing. This causes the issue.

**Comment by betteridiot** [comment]
As of the latest commit, this error shouldn't occur anymore (https://github.com/betteridiot/bamnostic/commit/cf1cc3da1bd7d82dc2bb71b27b3f0e981321ab68)
